packages/runtime-core/src/apiCreateApp.ts

```diff
@@ -224,7 +225,11 @@ export function createAppAPI<HostElement>(
         return app
       },
 
-      mount(rootContainer: HostElement, isHydrate?: boolean): any {
+      mount(
+        rootContainer: HostElement,
+        isHydrate?: boolean,
+        isSVG = false
```

Comment:
Couldn't the default value be removed here and in renderer.ts?

Reply:
Oh, sure